### ISSUE TITLE
Unhandled promise

### ISSUE THREAD

**Issue by benoitf** [crc-org/crc-extension#78]
Hello, FYI starting on a fresh OS install (no crc at all installed)

I got
![image](https://user-images.githubusercontent.com/436777/236140030-238c10a2-64c0-4ad6-bece-8a1ac03e01d5.png)

Upon the activation of the extension

**Comment by praveenkumar** [comment]
This looks like as expected because the those `sock` only available after `setup` so once you install the extension then you need to install the `crc` otherwise all the calls for status go through those sock and if they are not available, you will get this. May be in our extension codebase we need to make sure not make call to those socket until they are not available.

**Comment by benoitf** [comment]
I mean, extension should catch the error (to avoid the unhandled promise rejection)

There are probably missing `await` instructions

**Comment by gbraad** [comment]
Right. They should be handled... if not available, this should result in a no-op.
